src/irmin-pack/irmin_pack.ml

```diff
@@ -448,6 +448,11 @@ struct
         Contents.CA.close (contents_t t) >>= fun () ->
         Node.CA.close (snd (node_t t)) >>= fun () ->
         Commit.CA.close (snd (commit_t t)) >>= fun () -> Branch.close t.branch
+
+      let ro_sync t =
+        Contents.CA.ro_sync (contents_t t);
+        Node.CA.ro_sync (snd (node_t t));
+        Commit.CA.ro_sync (snd (commit_t t))
```

Comment:
what happens in the case where the 3 stores are actually the same store? Do we have a way to make this more efficient?

Reply:
I left only one `ro_sync` call and added a comment.